### ISSUE TITLE
error executing istioctl: exit status 64, Error: unknown flag: --filename

### ISSUE THREAD

**Issue by lgc0313** [tetratelabs/getmesh#72]
getmesh  version
```
getmesh version: 1.1.3
active istioctl: 1.11.3-tetrate-v0
no running Istio pods in "istio-system"
1.11.3-tetrate-v0
```
getmesh istioctl install --set profile=demo --filename istio-2-gw.yaml

```
Execute istioctl with given arguments where the version of istioctl is set by "getsitio fetch or switch"

Usage:
  getmesh istioctl <args...> [flags]

Examples:
# install Istio with the default profile
getmesh istioctl install --set profile=default

# check versions of Istio data plane, control plane, and istioctl
getmesh istioctl version

Flags:
  -h, --help   help for istioctl

Global Flags:
  -c, --kubeconfig string   Kubernetes configuration file
```

**Comment by boeboe** [comment]
I have the same issue

**Comment by psbrar99** [comment]
@lgc0313 @boeboe : this is addressed in new release of getmesh cli ( 1.1.4). Please update the getmesh cli to newer version.
